src/lv_core/lv_indev.c

```diff
@@ -695,50 +692,50 @@ static void indev_button_proc(lv_indev_t * i, lv_indev_data_t * data)
  */
 static void indev_proc_press(lv_indev_proc_t * proc)
 {
-    lv_obj_t * pr_obj = proc->types.pointer.act_obj;
+    lv_obj_t * indev_obj_act = proc->types.pointer.act_obj;
```

Comment:
remove the `lv_obj_t *`, you aren't using the global variable right now.

Reply:
I've also noticed it and already fixed here: https://github.com/littlevgl/lvgl/commit/5803fa621465b2600fcc90a53770fb336e1ef503